shell/platform/windows/win32_flutter_window_unittests.cc

```diff
@@ -27,9 +30,9 @@ namespace {
 // given.
 static LPARAM CreateKeyEventLparam(USHORT ScanCode,
                                    bool extended = false,
+                                   bool PreviousKeyState = 1,
```

Comment:
Probably these args should all have the correct case to match our style (lower_snake_case).  I think I copied the names from the Microsoft docs and forgot to fix that.

And this particular one might want to be called `was_down` to match the corresponding information on the keyboard hook.